### PR TITLE
Command line plugin and gem install/uninstall outside cluster

### LINKED ISSUE #520: Easily Load Plugins from Host Command Line

### PR DESCRIPTION
**Pull request by ryanmelt**
closes #520

### CODE REVIEW

File: openc3-cosmos-cmd-tlm-api/app/controllers/process_status_controller.rb [hunk 1]
```diff
@@ -31,6 +31,8 @@ def show
     return unless authorization('system')
     if params[:id].downcase == 'all'
       render :json => @model_class.all(scope: params[:scope])
+    elsif params[:id].split('__').length > 1
+      render :json => @model_class.get(name: params[:id], scope: params[:scope])
```
Comment (jmthomas) [inline openc3-cosmos-cmd-tlm-api/app/controllers/process_status_controller.rb:35]: What are we checking for here?

**Review by jmthomas — APPROVED** [review]
Looks good!

File: openc3-cosmos-cmd-tlm-api/app/controllers/process_status_controller.rb (hunk 1 above)
Comment (ryanmelt) [reply openc3-cosmos-cmd-tlm-api/app/controllers/process_status_controller.rb:35]: Process names have the form of hostname__pid

**Comment by tj-scherer** [comment]
How would one go about using this feature? I see that the following variables need to be declared in the .env file:
OPENC3_DEVEL
OPENC3_API_HOSTNAME
OPENC3_API_PORT
OPENC3_API_PASSWORD

How should each of these be set?

**Comment by ryanmelt** [comment]
From the release notes:
```
gem install openc3
openc3cli load /path/to/myplugin.gem
openc3cli unload myplugin.gem__12345667
openc3cli geminstall my.gem
openc3cli gemuninstall my.gem
```

Those environment variables should be set on your host, not the .env file.  Also, You probably don't want OPENC3_DEVEL.

**Pull request merged**